lib/src/accordion-group/AccordionGroup.tsx

```diff
@@ -73,79 +66,47 @@ DxcAccordionGroup.Accordion = AccordionGroupAccordion;
 
 const calculateWidth = (margin) => `calc(100% - ${getMargin(margin, "left")} - ${getMargin(margin, "right")})`;
 
-const AccordionGroupContainer = styled.div`
-  display: inline-block;
+const AccordionGroupContainer = styled.div<{ margin: Space | Margin; disabled: boolean }>`
```

Comment:
I think separating the types in a variable would be cleaner.

Reply:
Done 😉